packages/react/src/components/Menu/Menu.stories.js

```diff
@@ -45,10 +45,15 @@ export const Playground = (args) => {
   const selectableOnChange = action('onChange (MenuItemSelectable)');
   const radioOnChange = action('onChange (MenuItemRadioGroup)');
 
-  const target = document.getElementById('storybook-root');
+  const menuTarget =
+    document.getElementById(args.menuTarget) ||
+    document.getElementById('storybook-root');
```

Comment:
This can be removed, we don't typically expose element props like this within the actions pane.

Reply:
removed it also disabled menuTarget in actions as per our prior review